Makefile

```diff
@@ -80,7 +128,12 @@ build: build-yarn
 
 .PHONY: build-solidity-types
 build-solidity-types: ## generate Solidity typings
-	yarn workspace @hoprnet/hopr-ethereum run build:sol:types
+	echo "Foundry create binding"
+	$(MAKE) -C packages/ethereum/contracts/ overwrite-sc-bindings
+# Change git = "http://..." into version = "1.0.2"
+	sed -i -e 's/https:\/\/github.com\/gakonst\/ethers-rs/1.0.2/g' ${CURDIR}/packages/ethereum/crates/bindings/Cargo.toml
```

Comment:
Please undo these changes at the end of the target.

Reply:
Why should undo those changes?

Reply:
Ah, this "fixes" the generated files. In that case we can keep it as is.